es-vue-base/src/lib-components/EsReview.vue

```diff
@@ -124,6 +124,14 @@ export default {
             required: true,
             type: String,
         },
+        /**
+         * Updated Comment
+         */
```

Comment:
I suspect this is a case where CSM may be the only vertical using this prop. It might be worth putting more explanation into what it is.


```suggestion
        /**
         * Updated Comment
         * 
         * Used in es-cdgm when a user changes 
         * their original comment this field reflects
         * the updated comment.
         */
```